.github/workflows/check_cmake_format.yml

```diff
@@ -0,0 +1,17 @@
+name: check-cmake-format
+on: [pull_request, push]
+
+jobs:
+  build:
+    name: check-cmake-format
+    runs-on: ubuntu-20.04
+    # Run on external PRs, but not internal PRs as they'll be run by the push
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    steps:
+        - name: Fetch repository
+          uses: actions/checkout@v1
```

Comment:
Any particular reason for pinning an old version? IIUC the latest is v3.

Reply:
updated. thanks